### ISSUE TITLE
Default library does not match language settings

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix#373]
There are too many zim files available for comfortable or relevant browsing. When in the library, the application should by default offer content in the language Kiwix is set in (e.g. Kiwix is in French -> files shown in download library are in French)
![language setting](https://cloud.githubusercontent.com/assets/21675885/20705553/a26e8f98-b623-11e6-8059-0ee4160150a8.png)

**Comment by kelson42** [comment]
Agree, ticket should be kept to remember for new linux/windows/osx implementation.
